modules/react/common/stories/Responsive.stories.mdx

```diff
@@ -0,0 +1,333 @@
+import {Meta, ArgsTable} from '@storybook/addon-docs';
+import {Box} from '@workday/canvas-kit-react/layout';
+//
+import {
+  useResizeObserver,
+  ResponsiveContextProvider,
+  useResponsiveStyles,
+} from '@workday/canvas-kit-react/common';
+import {ResponsiveContainer} from './examples/ResponsiveStyles';
+import {ResponsiveViewport} from './examples/ResponsiveViewport';
+
+<Meta title="Hooks and Utilities/Responsive Styling" />
+
+# Responsive Styling
+
+Responsive styling is an approach to web development or a set of best practices, used to create a
+layout that can respond to the device being used to view the content.
```

Comment:
do we wanna say device or maybe size of the view port? or screen size?

Reply:
...respond to the device's screen size that is being used. 

Thoughts?

Reply:
Updated. 

...respond to the device's viewport size